### ISSUE TITLE
Accessibility focus first element doesn't work if no focusable element

### ISSUE THREAD

**Issue by danieliser** [PopupMaker/Popup-Maker#943]
The issue seems 2 fold.

1. The first focus element search excludes the close button.
2. The event seems to occur beforeOpen vs afterOpen
